prow/data-plane/values.yaml

```diff
@@ -0,0 +1,37 @@
+region: 'us-west-2'
+
+crier:
+  serviceAccount:
+    name: ''
+
+deck:
+  serviceAccount:
+    name: ''
+
+hook:
+  serviceAccount:
+    name: ''
+
+horologium:
+  serviceAccount:
+    name: ''
+
+prowControllerManager:
+  serviceAccount:
+    name: ''
+
+sinker:
+  serviceAccount:
+    name: ''
```

Comment:
Will this empty make SA name default to 'sinker' in the manifest file?

Reply:
Yeah in all of the service account name fields we default every name to the name of the deployment - that's the standard for Prow.